packages/database/src/migrations/20210318031303-UseTupaiaServiceInStrive-modifies-data.js

```diff
@@ -0,0 +1,279 @@
+      aggregationEntityType: 'village',
+    },
+    PG_Strive_PNG_Weekly_mRDT_Positive: { dataSourceEntityType: 'case' },
+    PG_Strive_PNG_Positive_RDT_By_Result_Over_Time: { dataSourceEntityType: 'case' },
+    PG_Strive_PNG_K13_PCR_Results: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'facility',
+    },
+    SCRF_RDT_Positive_Results: { dataSourceEntityType: 'case' },
+  },
+  overlays: {
+    PG_STRIVE_K13_C580Y_Positive_Count: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'facility',
+    },
+    PG_STRIVE_K13_C580Y_Positive: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'facility',
+    },
+    PG_STRIVE_PF05_Positive: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'facility',
+    },
+    PG_STRIVE_PM05_Positive: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'facility',
+    },
+    PG_STRIVE_PO05_Positive: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'facility',
+    },
+    PG_STRIVE_PV05_Positive: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'facility',
+    },
+    PG_STRIVE_QMAL05_Positive: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'facility',
+    },
+    STRIVE_CRF_Febrile_Cases_Radius: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'facility',
+    },
+    STRIVE_CRF_mRDT_Tests_Radius: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'facility',
+    },
+    STRIVE_CRF_Positive_Mixed_Percentage: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'facility',
+    },
+    STRIVE_CRF_Positive_Non_Pf_Percentage: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'facility',
+    },
+    STRIVE_CRF_Positive_Pf_Percentage: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'facility',
+    },
+    STRIVE_CRF_Positive: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'facility',
+    },
+    STRIVE_FIS_Village_Number_Reported_Cases_In_Week: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'village',
+    },
+    STRIVE_FIS_Village_Percent_mRDT_Positive_In_Week: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'village',
+    },
+    STRIVE_FIS_Village_Percent_mRDT_Positive_Mixed_In_Week: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'village',
+    },
+    STRIVE_FIS_Village_Percent_mRDT_Positive_Non_PF_In_Week: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'village',
+    },
+    STRIVE_FIS_Village_Percent_mRDT_Positive_PF_In_Week: {
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'village',
+    },
+    STRIVE_WTF_Consultations_Radius: { dataSourceEntityType: 'facility' },
+    STRIVE_WTF_mRDT_Tests_Radius: { dataSourceEntityType: 'facility' },
+    STRIVE_WTF_Positive: { dataSourceEntityType: 'facility' },
+  },
+};
```

Comment:
I didn't really go through this list to check of the `dataSourceEntityType` or `aggregationEntityType` are correct, but I assume you've already smoke tested them 👍

Reply:
Good call I believe, yes I've smoke tested them, and we'll do some thorough regression testing which will hopefully catch anything that may be wrong!